### PR TITLE
gives preference to user if they want to automatch or not

### LINKED ISSUE #4722: « Auto-match candidates with high confidence » is deceitful and should not be pre-check
In the Reconciliation dialog, the checkbox « Auto-match candidates with high confidence » is pre-checked when entering the dialog.

Since the confidence from the Recon service is not always useful (it can say 100% to many items even when some should be defraded, user don't really know what the number means and exactly how it is given). Because of that, I think the checkbox should be un-cheked by default. A preference could be create to control the default value of that control.

Since this has a direct impact on data pushed to Wikidata, and I highly raise the possibility of bad data being pushed, I consider this a bug.

### To Reproduce
1. Open the reconciliation dialog

### Current Results
The checkbox « Auto-match candidates with high confidence » is pre-checked.

### Expected Behavior
The checkbox « Auto-match candidates with high confidence » is unchecked when opening the dialog.

### Screenshots
image

### Versions
 - Operating System: N/A
 - OpenRefine: v3.5.x and below.

### PR DESCRIPTION
**Pull request by ayushrai206**
Fixes #4722 

Changes proposed in this pull request:
- gives preference to user if they want to automatch candidates with high confidence or not, keeping it true by default .

### CODE REVIEW

File: main/webapp/modules/core/scripts/reconciliation/standard-service-panel.js [hunk 1]
```diff
@@ -337,7 +337,7 @@ ReconStandardServicePanel.prototype.start = function() {
         identifierSpace: this._service.identifierSpace,
         schemaSpace: this._service.schemaSpace,
         type: (type) ? { id: type.id, name: type.name } : null,
-        autoMatch: this._elmts.automatchCheck[0].checked,
```
Comment (Abbe98) [inline main/webapp/modules/core/scripts/reconciliation/standard-service-panel.js:340]: This should remain as it just gets the value of "checked" from the DOM?

File: main/webapp/modules/core/scripts/reconciliation/standard-service-panel.js [hunk 2]
```diff
@@ -87,7 +87,7 @@ ReconStandardServicePanel.prototype._constructUI = function() {
   var self = this;
   this._panel = $(DOM.loadHTML("core", "scripts/reconciliation/standard-service-panel.html")).appendTo(this._container);
   this._elmts = DOM.bind(this._panel);
-  
+  this._elmts.automatchCheck[0]=JSON.parse(Refine.getPreference("ui.reconciliation.automatch", true));
```
Comment (Abbe98) [inline main/webapp/modules/core/scripts/reconciliation/standard-service-panel.js:90]: I expect this to still set the `checked` attribute.

**Review by Abbe98 — CHANGES_REQUESTED** [review]
This only appears to be true by default because that is set in the HTML, meanwhile the new code does not interact with the `checked` attribute so it does not seem to have the intended effect.

**Review by Abbe98 — APPROVED** [review]
Nicely done!

**Comment by wetneb** [comment]
done by https://github.com/OpenRefine/openrefine.org/pull/209

**Comment by ayushrai206** [comment]
> Could it be possible to have an accompanying documentation update? Otherwise users are unlikely to discover it.
yes, working on it.

**Comment by antoine2711** [comment]
@ayushrai206 : @Abbe98 took care of it. Check the previous comment by @wetneb.

Thank you very much for your work and congrat for this achievement.

Best Regards, Antoine

**Comment by Abbe98** [comment]
Sorry! I already had a draft...

**Pull request merged**